.github/workflows/golangci-lint.yml

```diff
@@ -2,6 +2,8 @@ name: golangci-lint
 on:
   pull_request:
     types: [opened, edited, synchronize, reopened]
```

Comment:
There are a few more 1.17's in .golangci.yml

(but we have to wait with merge until the next golangci-lint release)

Reply:
Thank you! It's been updated.